src/applications/disability-benefits/686c-674/tests/e2e/686-test-data.json

```diff
@@ -3,11 +3,7 @@
     "optionsSelection": {
       "addChild": true,
       "addSpouse": true,
-      "reportDivorce": true,
```

Comment:
Does master need to be merged into your branch? This reverts work that was merged in this pull request - https://github.com/department-of-veterans-affairs/vets-website/pull/12854/files

Reply:
no it was a bad commit fixing it now